warehouse/ingest-core/src/main/java/datawave/ingest/data/config/ingest/BaseIngestHelper.java

```diff
@@ -551,6 +551,11 @@ public void addIndexOnlyField(String fieldName) {
         indexOnlyFields.add(fieldName);
     }
     
+    @Override
+    public void addShardExclusionField(String fieldName) {
```

Comment:
Lets use this method around line 393

Reply:
Considering that we have an addIndexOnlyField method and we don't call it in the equivalent spot at 411, I didn't think it mattered.